### PR TITLE
Add XML example

### PR DESCRIPTION
**Pull request by nspyke**

### CODE REVIEW

File: core/swagger.md [hunk 1]
```diff
@@ -184,6 +184,29 @@ resources:
               format: date-time
 ```
 
+Or in XAML:
```
Comment (alanpoulain) [inline core/swagger.md:187]:
```suggestion
Or in XML:
```

File: core/swagger.md [hunk 2]
```diff
@@ -184,6 +184,29 @@ resources:
...
+    <resource class="App\Entity\Product">
+        <property name="name">
+            <attribute name="openapi_context">
+                <attribute name="type">type</attribute>
+                <attribute name="enum">
+                    <attribute>one</attribute>
+                    <attribute>two</attribute>
+                </attribute>
+                <attribute name="example">one</attribute>
+            </attribute>
```
Comment (alanpoulain) [inline core/swagger.md:204]: It misses the `timestamp` attribute.

**Comment by nspyke** [comment]
Thanks @alanpoulain - have made the updates

**Comment by soyuka** [comment]
thanks @nspyke

**Pull request merged**